x/precisebank/keeper/view_integration_test.go

```diff
@@ -129,3 +129,30 @@ func (suite *viewIntegrationTestSuite) TestKeeper_SpendableCoin() {
 		})
 	}
 }
+
+func (suite *viewIntegrationTestSuite) TestKeeper_HiddenReserve() {
+	// Reserve balances should not be shown to consumers of x/precisebank, as it
+	// represents the fractional balances of accounts.
+
+	moduleAddr := authtypes.NewModuleAddress(types.ModuleName)
+	addr1 := sdk.AccAddress{1}
+
+	// Make the reserve hold a non-zero balance
+	// Mint fractional coins to an account, which should cause a mint of 1
+	// integer coin to the reserve to back it.
+	suite.MintToAccount(addr1, sdk.NewCoins(sdk.NewCoin(types.ExtendedCoinDenom, sdk.NewInt(1000))))
```

Comment:
can you mint some of another denom to this address & write a test confirming that that denom is unaffected?

on mainnet, we never expect any other denom, but feels worthwhile to verify that the overridden zero balance is only for akava

Reply:
correction / update: we _do_ expect another denom. it will have ukava if the sum of all the different fractional balances is greater than the conversion factor.

so the ask: can you also mint ukava to the module in this test & verify that it correctly responds with a non-zero balance when queried?